src/game/engine/techno.cpp

```diff
@@ -1633,20 +1633,45 @@ VisualType TechnoClass::Visual_Character(BOOL flag) const
 }
 
 /**
- *
+ * Kill cargo, record the kill for the killer
  *
  */
 void TechnoClass::Kill_Cargo(TechnoClass *object)
 {
     while (m_Cargo.Has_Cargo()) {
-        FootClass *ptr = m_Cargo.Detach_Object();
```

Comment:
Its unnessesary to split the assignment from the declaration of the variable in the code as it currently stands, please revert.